### ISSUE TITLE
Nothing appearing in "observed outputs" pane version 0.6

### ISSUE THREAD

**Issue by bennyncc** [nccgroup/tracy#80]
Updated to newest Tracy, when payloads are generated in Chromium, I see them appear in the `injection points` pane, but nothing appears in the `observed outputs` pane.  In the terminal I'm running Tracy on, I see a lot of these errors:

`[ERROR]:2019/02/07 10:50:35 proxy.go:415: dial tcp: lookup peasenvqmu on 127.0.0.53:53: server misbehaving`

and

`[ERROR]:2019/02/07 14:12:08 event.go:28: table dom_contexts has no column named reason`

**Comment by heathj** [comment]
Sounds like you are using an old database. Try:

~~~
rm ~/.tracy/*.db
~~~

and rebooting tracy.

**Comment by heathj** [comment]
hmm was peasenvqmu a tracy payload?

**Comment by bennyncc** [comment]
> hmm was peasenvqmu a tracy payload?

No, but there are many similar lines, it looks like it would be:

```
[ERROR]:2019/02/07 14:17:17 proxy.go:415: dial tcp: lookup kmdzlqnlnogkb on 127.0.0.53:53: server misbehaving
[ERROR]:2019/02/07 14:17:17 proxy.go:415: dial tcp: lookup gijeoygegw on 127.0.0.53:53: server misbehaving
[ERROR]:2019/02/07 14:17:17 proxy.go:415: dial tcp: lookup bmzjjqhbjzccpun on 127.0.0.53:53: server misbehaving
```
